mora/service/orgunit.py

```diff
@@ -223,8 +224,7 @@ def get_one_orgunit(c, unitid, unit=None,
             'invalid details {!r}'.format(details),
         )
 
-    if validity is not None:
-        r[keys.VALIDITY] = validity
+    r[keys.VALIDITY] = validity or common.get_effect_validity(validities[0])
```

Comment:
Probably somewhat unrelated, but what is the point of the `validity` parameter to this function? Just to be used here?

Reply:
> Probably somewhat unrelated, but what is the point of the `validity` parameter to this function? Just to be used here?

We use it for the `/service/ou/$UUID/details/org_unit` calls — if you remove it, those tests fail.